lib/image-stream/supported.js

```diff
@@ -56,5 +56,9 @@ module.exports = [
   {
     extension: 'raw',
     type: 'image'
+  },
+  {
+    extension: 'dmg',
+    type: 'image'
```

Comment:
@jhermsmeier From what you've been saying, I guess this is actually a 'compressed image'? Do we need to make that distinction? @jviotti What are these `type` labels used for?